### DISCUSSION TITLE
Desactivar la vista dinámica de Blogger

### DISCUSSION THREAD

**Discussion by giscus** [zkreations/data#11] [Announcements]
# Desactivar la vista dinámica de Blogger

Este modo siempre está habilitado, aún cuando no lo hayas activado en el panel de administración, por eso es accesible en todos los blogs de Blogger

https://www.zkreations.com/2022/12/desactivar-la-vista-dinamica-de-blogger.html

**Comment by TakoMics** [comment]
Excelente aporte, ni enterado de esto.

Todo un diseño arcáico y retro esa vista dinámica.

**Reply by danieIabel** [reply]
Asi es amigo, a mi tampoco me agrada, por eso siempre la he tenido deshabilitada de esta manera, pero al revisar blogs por internet, he notado que casi nadie estaba enterado.

**Comment by cristianime** [comment]
Casi me muero con lo de la ruta 😂 gracias por el artículo Daniel, ya estoy por quitarlas pero una pregunta. No afecta al seo? No sería mejor bloquear esto con robots.txt? 🤔

**Reply by danieIabel** [reply]
Que tal amigo, en principio no se deberían indexar esas rutas ya que si revisas el codigo fuente notarás que tienen una etiqueta "nofollow" en la cabecera. Pero es verdad que google a veces la ignora y por error se pueden colar, no es lo habitual claro pero puede ocurrir. De todas formas, el objetivo de este artículo es desactivarlas por completo y esto no se logra con el archivo robots.txt, el cual dicho sea de paso yo no recomiendo modificar sin que estés muy seguro de que sabes lo que haces.

Sobre si afecta al SEO, yo llevo usando estas redirecciones desde hace muchos años, y no he notado nada negativo relacionado a ellas, más bien, quitarlas me da la sensación de que el blog es más profesional.

**Reply by cristianime** [reply]
Gracias por la explicacion daniel muy completo 👍
